### DISCUSSION TITLE
Prevent styles compilation when importing from vuetify/lib

### DISCUSSION THREAD

**Discussion by Maxim-Mazurok** [vuetifyjs/vuetify#14233] [Q&A] [answered]
I'm using Vuetify with TSX, so my components look like this:

```tsx
import * as tsx from "vue-tsx-support";
import { VNode } from "vue";
import { VBtn } from "vuetify/lib";

export default tsx.component({
  name: "Dashboard",
  render(): VNode {
    return (
      <div>
        <h1>Dashboard will be here</h1>
        <VBtn>some button</VBtn>
      </div>
    );
  },
});
```

How can I prevent sass/scss sources from being imported/compiled? I just want to use "vuetify/dist/vuetify.min.css" to speed up the build process.

I've tried `file-replace-loader` and webpack ignore plugin, but they didn't work. Should I keep trying, or there's a better way?...

**Comment by KaelWD** [comment, marked as answer]
https://github.com/vuetifyjs/vuetify-loader/blob/68bb4aac414eb2fc77fe8bd21cdca5c1a149715e/packages/vuetify-loader/src/plugin.ts#L73-78

**Reply by Maxim-Mazurok** [reply]
I can see that this is available in `@next` version, but I can't install `@next` version because I use Vue 2. Are there any plans to release this feature for Vue 2?

**Reply by Maxim-Mazurok** [reply]
Interestingly, even after I added this:
```js
module.exports = {
  chainWebpack(config) {
    config.module
      .rule("bla-bla")
      .enforce("pre")
      .test(/\.(css|scss|sass)$/)
      .use("null-loader")
      .loader("null-loader")
      .end();
  },
};
```

I can still see these in the build log:
```
56 │ $container-padding-x: $grid-gutter / 2 !default;
   │                       ^^^^^^^^^^^^^^^^
   ╵
    node_modules/vuetify/src/styles/settings/_variables.scss 56:23  @import
    node_modules/vuetify/src/styles/settings/_index.sass 1:9        @import
    node_modules/vuetify/src/styles/styles.sass 2:9                 @import
    node_modules/vuetify/src/styles/main.sass 2:9                   root stylesheet

DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
```

Even though all resulting CSS files are 0kb...

**Reply by Maxim-Mazurok** [reply]
see https://github.com/vuetifyjs/vuetify-loader/issues/209

**Reply by KaelWD** [reply]
Apparently it's a bit more complicated with sass: https://github.com/vuetifyjs/vuetify/issues/10048#issuecomment-916190274
